templates/observations.html

```diff
@@ -349,9 +346,6 @@
             <div class="tab-pane {% if selected_chart == "heat_map" %} active {% else %} fade {% endif %}" id="heat_map">
               {% include "_star_chart_heat_map.html" %}
             </div>
-            <div class="tab-pane {% if selected_chart == "chart_v3" %} active {% else %} fade {% endif %}" id="chart_v3">
```

Comment:
How about logic of chart_v3 in backend? I think we can delete it too.

Reply:
I thought about it but the logic is the same like in star chart 2 so I can't remove it.